### ISSUE TITLE
[Feature Request] More intuitive way to adjust/limit amount of videos to be downloaded

### ISSUE THREAD

**Issue by suckerfish** [bbilly1/tubearchivist#189]
I think when someone first adds a subscription it should be very easy to tell TA that you want to get the entire channel or not _or_ give some nice controls about which videos exactly they will be getting when they subscribe. Maybe radio buttons or checkboxes or something "Download entire channel?" and "Specify date range for which videos you want" or "Get X number of recent videos?" 

The only way to do this currently is to adjust a global setting about how far back TA looks for **all** subscriptions.

**Comment by bbilly1** [comment]
Yes, but I'd say this would better fit on the downloads page than on the subscribe page. these are different tasks that run in the background. 

Then on the subscribe page, there could be a mention about how many videos this will add next time you rescan.

**Comment by BlipRanger** [comment]
Yeah, regarding this; what is the correct way to instruct TA to archive an entire channel? Just bumping the limit up really high?
